modules/tinymce/src/themes/silver/main/ts/ui/menus/item/build/CardMenuItem.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { AlloyComponent, AlloySpec, Behaviour, Disabling } from '@ephox/alloy';
+import { Menu } from '@ephox/bridge';
+import { Arr, Optional } from '@ephox/katamari';
+import { UiFactoryBackstageShared } from 'tinymce/themes/silver/backstage/Backstage';
+import { renderItemDomStructure } from 'tinymce/themes/silver/ui/menus/item/structure/ItemStructure';
+import ItemResponse from '../ItemResponse';
+import { renderContainer, renderDescription, renderHtml, renderImage } from '../structure/ItemSlices';
+import { replaceText } from './AutocompleteMenuItem';
+import { buildData, renderCommonItem } from './CommonMenuItem';
+
+export interface CardExtras {
+  itemBehaviours?: Array<Behaviour.NamedConfiguredBehaviour<Behaviour.BehaviourConfigSpec, Behaviour.BehaviourConfigDetail>>;
+  title: { // Extras specific to title components
```

Comment:
Maybe move this comment to the line above? as this really applies to the whole title block.